README.md

```diff
@@ -57,52 +54,62 @@ Authorino complies with the [3scale Ostia architecture](https://github.com/3scal
+
+_Identity verification_ can be, for example, verifying a provided JSON Web Token (JWT) previously issued by an OIDC-compliant authentication server and provided in the `Authorization` header of the HTTP request. Authorino will auto-discover the OIDC configuration from the settings, verify the token signature and validity in time.
+
+_Ad-hoc authorization metadata_ can be the fetching of OIDC user info (based on the same configuration used for the identity before) and/or about fecthing resource data from a UMA-compliant server that knows about the resources.
+
+_Policy enforcement_ is where Authorino connects to one or more authorization policy evaluation services (or Policy Decision Points (PDP)) that will ultimately decide on whether the HTTP request should be authorized or not, given the known gathered information about the identity, the origin of the request, the requested resource and, of course, the configured policies.
+
+_Identity verification_ and _Ad-hoc authorization metadata_ steps add info for the _Policy enforcement_, meaning API developers can write authorization policies trusting on information available about the requestor's identity (a decoded JWT, for example) and on the metadata added ad-hoc to the authorization payload (e.g. specific user info or data about the requested resource). Of course, if Authorino fails to verify the user's identity in the _Identity verification_ step, the request is rejected without further development of the other two steps.
+
 ## Features
 
-Authorino will support at least 3 different authentication methods (i.e., OIDC, user/passwd and mTLS), plus ad-hoc
-additions to the authorization payload (e.g., user info, resource metadata, web hooks), and combination of multiple
-authorization services (JWT claims, OPA, Keycloak), all driven by configuration. Authorino will also handle caching of
-user credentials, permissions, revocations, etc.
+Once ready, Authorino will support at least 3 different authentication methods (i.e., OIDC, user/passwd and mTLS), plus ad-hoc additions to the authorization payload (e.g., user info, resource metadata, web hooks), and combination of multiple authorization services (JWT claims, OPA, Keycloak). Authorino will also handle caching of user credentials, permissions, revocations.
+
+Here's a list of features related to each of Authorino's 3 core steps and supporting features. Features listed as "PoC" mean they are already implemented and ready to use in the current stage of Authorino, which is still in proof of concept. "Planned" are the features that are part of the original proposal of Authorino but not yet in PoC. "In analysis" are suggested features that may require extra effort to be implemented and therefore are still being analyzed regarding viability or design.
```

Comment:
How about converting this long sentence into a legend with bullets ( * PoC: ... , * Planned: ... , * In analysis: ... ) under the table, for easier scanning?